app/helpers/mobile_helper.rb

```diff
@@ -39,6 +39,21 @@ def mobile_like_icon(post)
     end
   end
 
+  def mobile_like_comment_icon(comment)
+    puts "Mobile like on comment!"
+    if current_user && current_user.liked?(comment)
```

Comment:
Style/SafeNavigation: Use safe navigation (`&.`) instead of checking if an object exists before calling the method.